pkg/cmd/pipelinerun/logs.go

```diff
@@ -0,0 +1,220 @@
+// Copyright © 2019 The Knative Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelinerun
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/cmd/taskrun"
+	"github.com/tektoncd/cli/pkg/logs"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	msgPipelineNotFoundErr = "Error in retrieving Pipeline"
+	msgPRNotFoundErr       = "Error in retrieving PipelineRun"
+)
+
+type taskRunMap map[string]*v1alpha1.PipelineRunTaskRunStatus
+
+type PipelineRunLogs struct {
+	Name    string
+	Ns      string
+	Clients *cli.Clients
+}
+
+type taskRun struct {
+	name   string
+	task   string
+	status v1alpha1.TaskRunStatus
+}
+
+type LogOptions struct {
+	taskrun.LogOptions
+	OnlyTasks []string
+}
+
+func NewPipelineRunLogs(pipelineRunName string, namespace string, clients *cli.Clients) *PipelineRunLogs {
+	return &PipelineRunLogs{
+		Name:    pipelineRunName,
+		Ns:      namespace,
+		Clients: clients,
+	}
+}
+
+func logCommand(p cli.Params) *cobra.Command {
+	var (
+		follow    bool
+		allSteps  bool
+		onlyTasks []string
```

Comment:
Shouldn't this just be `tasks`, as the tasks I want to show the log for ? (and if it's empty, it means all)